src/pysight/nd_hist_generator/movie.py

```diff
@@ -143,9 +144,8 @@ def __determine_outputs(self) -> Tuple[List[Callable], List[Callable]]:
                 funcs_to_execute_end.append(self.__save_stack_at_once)
             if "summed" in self.outputs:
                 funcs_to_execute_end.append(self.__save_summed_at_once)
-            # TODO: Figure out what to do here
-            # if "flim" in self.outputs:
-            #     funcs_to_execute_end.append(self.__save_flim_at_once)
+            if "flim" in self.outputs:
+                funcs_to_execute_end.append(self.__save_flim_at_once)
```

Comment:
The logic here is wrong but it's really not your fault, it's a messy business overall.

Reply:
I'll handle it tomorrow.